js/angular/app/scripts/modules/userdata/change-timezone-controller.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+angular.module('transitIndicators')
+.controller('OTIUserdataChangeTimezoneController',
+            ['config', '$modalInstance', '$scope', '$state', '$cookieStore', 'OTILocalization', 'timezones',
+            function (config, $modalInstance, $scope, $state, $cookieStore, OTILocalization, timezones) {
+
+    $scope.selectTimezone = function(timezone) {
+        OTILocalization.setTimeZone(timezone.zone).then(function(response) {
+            $modalInstance.close();
```

Comment:
Is it a requirement that a user select a timezone to close the modal? If so, might want to add a comment here stating that so we don't accidentally change it later, and also should set the backdrop: 'static' parameter in the associated `$modal.open` call above.